src/haipa-zero/ZeroContainerExtensions.cs

```diff
@@ -25,12 +28,27 @@ public static void Bootstrap(this Container container, string[] args)
             container.HostModule<VmHostAgentModule>();
             container.HostModule<ControllerModule>();
 
+            container.CreateSsl(certOptions =>
```

Comment:
This part is missleading. 
For a reader this looks like a container registration without side effects. Actually it executes the commands to setup SSL. A better place would be before the container setup in Program class of haipa-zero.